hermes-api/src/main/java/pl/allegro/tech/hermes/api/Subscription.java

```diff
@@ -222,129 +233,20 @@ public boolean isActive() {
 
     public Subscription anonymizePassword() {
         if (getEndpoint().containsCredentials()) {
-            return Builder.subscription().withName(this.getName())
-                    .withDescription(this.description)
-                    .withTopicName(this.getTopicName())
-                    .withEndpoint(this.getEndpoint().anonymizePassword())
-                    .withState(this.getState())
-                    .withSubscriptionPolicy(this.getSerialSubscriptionPolicy())
-                    .withSubscriptionPolicy(this.getBatchSubscriptionPolicy())
-                    .withTrackingEnabled(this.trackingEnabled)
-                    .withSupportTeam(this.supportTeam)
-                    .withContact(this.contact)
-                    .build();
+            return new Subscription(
+                    topicName,
+                    name,
+                    endpoint.anonymizePassword(),
+                    state,
+                    description,
+                    deliveryType == DeliveryType.SERIAL ? serialSubscriptionPolicy : batchSubscriptionPolicy,
```

Comment:
I think that SERIAL delivery is a better default and BATCH should be enabled explicitly in all cases...